client/src/format.ts

```diff
@@ -35,6 +35,26 @@ export function formatCountyASMState(state: County.ASMState): string {
     }
 }
 
+function formatFileUploadStatus(countyStatus: DOS.CountyStatus): string {
+// here we are either really failed or pending
+    if (_.get(countyStatus, 'ballotManifest.result.errorMessage') === undefined &&
+        _.get(countyStatus, 'cvrExport.result.errorMessage') === undefined) {
+        return 'File Upload In Progress...';
+    } else {
+        return 'File Upload Failed';
```

Comment:
These should read like all the other statuses - sentence-cased: `File upload in progress` (drop the ellipsis, since we use that for truncation), and `File upload failed`

Reply:
done